### PR TITLE
new split file tool

### PR DESCRIPTION
**Pull request by caleb-easterly**
Mentioned in #757 

@bgruening @jj-umn Can you take a look at this?

Bjoern, I didn't know if this was the best place for this tool - happy to move it somewhere else if that would be better.

### CODE REVIEW

File: tools/text_processing/split_file_to_collection/split_file_to_collection.xml [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+                </when>
+            </conditional>
+        </xml>
+    </macros>
+    <requirements>
+        <requirement type="package" version="3.5">python</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+        mkdir ./out &&
+        python $__tool_directory__/split_file_to_collection.py
```
Comment (bgruening) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.xml]: single quotes

File: tools/text_processing/split_file_to_collection/split_file_to_collection.xml [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+            #end if
+        #if ($split_parms.select_ftype == "tabular" and $split_parms.split_by.select_split_by == "row"):
+             --file_names '$split_parms.split_by.newfilenames'
+             --file_ext '$split_parms.select_ftype'
+        #end if
+        #if $split_parms.select_ftype != "tabular":
+            --file_names '$split_parms.newfilenames'
+            --file_ext '$split_parms.select_ftype'
+        #end if
+        > $log
```
Comment (bgruening) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.xml]: single-quotes

File: tools/text_processing/split_file_to_collection/split_file_to_collection.py [hunk 3]
```diff
@@ -0,0 +1,260 @@
+import argparse
```
Comment (bgruening) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.py]: shebang? /usr/bin/env python

File: tools/text_processing/split_file_to_collection/split_file_to_collection.xml [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+    ]]></command>
+    <inputs>
+        <conditional name="split_parms">
+            <param name="select_ftype" type="select" label="Select the file type to split">
+                <option value="mgf">MGF</option>
+                <option value="fastq">FASTQ</option>
```
Comment (bgruening) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.xml]: why are you separating here the different fastq types?

**Comment by bgruening** [comment]
I don't know where it is best place. Maybe even GalaxyP. I'm also fine leaving it here under bgruening or rnateam, your call :)

**Comment by caleb-easterly** [comment]
If you're okay with keeping it here, I think that would be best for us :)

**Comment by bgruening** [comment]
Thanks @caleb-easterly!

**Comment by caleb-easterly** [comment]
thank you, @bgruening!

**Pull request merged**